presto-tests/src/main/java/io/prestosql/tests/AbstractTestQueries.java

```diff
@@ -4617,6 +4618,10 @@ public void testAccessControl()
         assertAccessAllowed("SELECT name AS my_alias FROM nation", privilege("my_alias", SELECT_COLUMN));
         assertAccessAllowed("SELECT my_alias from (SELECT name AS my_alias FROM nation)", privilege("my_alias", SELECT_COLUMN));
         assertAccessDenied("SELECT name AS my_alias FROM nation", "Cannot select from columns \\[name\\] in table .*.nation.*", privilege("name", SELECT_COLUMN));
+        assertAccessDenied("SHOW CREATE TABLE orders", "Cannot show columns of table .*.orders.*", privilege("orders", SHOW_COLUMNS));
+        assertAccessAllowed("SHOW CREATE TABLE lineitem", privilege("orders", SHOW_COLUMNS));
+        assertAccessDenied("SHOW CREATE VIEW view_orders", "Cannot show columns of table .*.orders.*", privilege("view_orders", SHOW_COLUMNS));
```

Comment:
`of view`?

Maybe just drop `of table `.

Reply:
ok